packages/Select/src/select-value.scss

```diff
@@ -23,14 +23,14 @@
     --ids-select-value-padding: var(--hop-space-stack-sm) var(--hop-space-inline-md);
 
     /* Compact **/
-    --ids-select-value-padding-compact: 0.375rem var(--hop-space-inline-md);
+    --ids-select-value-padding-compact: calc(var(--hop-space-stack-sm) - var(--ids-select-broder-width) * 2) var(--hop-space-inline-md);
```

Comment:
`broder-width ` should be `border-width`

Reply:
good catch